projects/angular-test-app/tests/e2e/DigV2/ComplexFields/DataReference.spec.js

```diff
@@ -283,18 +279,18 @@ test.describe('E2E test', () => {
 
     let tableCell = table.locator('tbody >> tr >> td >> nth=1');
     // "---" should come at the top in the ascending order, since it's a Falsy value
-    await expect(await tableCell.textContent()).toBe('---');
+    expect(await tableCell.textContent()).toBe('---');
 
     await table.locator('div:has-text("Product Name") >> nth=0').click();
 
     tableCell = table.locator('tbody >> tr >> td >> nth=1');
-    // "Luxury Product" should be at the top in the descending order
-    await expect(await tableCell.textContent()).toBe('Luxury Product');
+    // "Red Item" should be at the top in the descending order
```

Comment:
Why did we replace `Luxury Product` with `Red Item`? Has the values in the DigV2 app changed?